app/validators/email_validator.rb

```diff
@@ -0,0 +1,9 @@
+class EmailValidator < ActiveModel::Validator
+  EMAIL_VALIDATION_ERROR_MESSAGE = "^Enter an email address in the correct format, like name@example.com".freeze
+
+  def validate(record)
+    if record.email.nil? || record.email.empty? || !record.email.match?(/@/)
+      record.errors[:email] << EMAIL_VALIDATION_ERROR_MESSAGE
```

Comment:
```suggestion
      record.errors[attribute] << EMAIL_VALIDATION_ERROR_MESSAGE
```

`:email` hard coding that is bad idea